internal/serverinstall/k8s.go

```diff
@@ -309,12 +263,13 @@ func (i *K8sInstaller) Install(
 
 		// Set the grpc address
 		grpcAddr = fmt.Sprintf("%s:%d", addr, grpcPort)
-		log.Info("server service ready", "addr", addr)
+		log.Info("server service ready: %s", addr)
```

Comment:
This unfortunately doesn't work with `hclog`. The log message here will say like `EXTRA_END VALUE` or something. Basically the hclog functions aren't formatters in the way `ui.Output` is so you'll still want `log.Info("msg", "key", value)`